src/sentry/pipeline/base.py

```diff
@@ -41,13 +47,13 @@ class Pipeline:
     using the ``update_config`` method.
     """
 
-    pipeline_name = None
-    provider_manager = None
-    provider_model_cls = None
+    pipeline_name: str
+    provider_manager: Any
```

Comment:
Could be worth defining some kind of interface here at some point, not necessarily this pr though. Seems like we just require that the manager implements a `.get` that returns a `PipelineProvider`